src/hooks/useSocial.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { SocialProvider } from '../components/SocialProvider';
+import { useSocial } from '@app/hooks';
+import { Account } from 'near-api-js';
+import createEphemeralAccount from '@test/helpers/createEphemeralAccount';
+
+jest.mock('@app/hooks', () => ({
+
+  useSocial: jest.fn(() => ({
+    social: {
+      async get({ keys }: { keys: string[] }) {
+        return keys.reduce((acc, key) => {
```

Comment:
`.reduce` allows you to declare the the type to cast, rather than using the `as` notation.

```ts
keys.reduce<Record<string, unknown>>((acc, key) => {
  acc[key] = { profile: { name: 'TestUser' } };
  return acc;
}, {});
```